common/djangoapps/student/views.py

```diff
@@ -893,6 +884,50 @@ def dashboard(request):
     return response
 
 
+@login_required
+def course_run_refund_status(request, course_id):
+    """
+    Get Refundable status for a course.
+
+    Arguments:
+        request: The request object.
+        course_id (str): The unique identifier for the course.
+
+    Returns:
+        Json response.
+
+    """
+
+    if course_id:
+        try:
+            course_key = CourseKey.from_string(course_id)
+            course_enrollment = CourseEnrollment.get_enrollment(request.user, course_key)
+
+        except InvalidKeyError:
+            logging.exception("An error as occured while getting refund status for a course")
```

Comment:
change the exception message to a more appropriate message related to course enrollment because if some issue comes and this exception is loged it will confuse debugging person to find which triggered this exception.